lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -135,7 +135,11 @@ void CPUBackend::save(Function *F, llvm::StringRef outputDir,
   BundleSaver(IR.get()).save(tgt, outputDir, networkName);
 }
 
-bool CPUBackend::isOpSupported(Kinded::Kind opKind, ElemKind elementTy) const {
+CostVal CPUBackend::getOpCost(Kinded::Kind opKind,
+                              llvm::ArrayRef<TypeRef> inputTypes,
+                              llvm::ArrayRef<TypeRef> outputTypes) const {
+  // TEMPORARY: Will remove once not a WIP. This is just so things compile.
+  auto elementTy = inputTypes[0]->getElementType();
```

Comment:
Do you plan on handling int16 and fp32 in the follow up?

Reply:
Yep! I will update the CPU and OpenCL backend's versions too in this PR. I first just wanted to make sure everyone agreed the Interpreter's `getOpCost()`/the general approach seemed reasonable.